terraform/Makefile

```diff
@@ -51,6 +51,7 @@ ifeq ($(AZUL_PRIVATE_API),1)
 	terraform apply -target aws_vpc_endpoint.indexer -target aws_vpc_endpoint.service
 endif
 	terraform apply
+	python $(project_root)/scripts/delete_lambda_versions.py
```

Comment:
I believe we agreed to delete old versions *before* the new version is published so that we keep the most recent old version around.